students/olszewski_bartosz/lesson_06_dictionaries/uppercase.py

```diff
@@ -0,0 +1,16 @@
+a = input()
+
+
+def capitalize(sentence):
+    for i in range(len(sentence)):
+        if i == 0:
+            b = ord(sentence[i]) - 32
+            print(chr(b), end='')
+        elif sentence[i - 1] == ' ':
```

Comment:
Consider to loop over words instead of checking for space in previous index

before:
for i in range(len(sentence)):

after;'
capitalized_words = []
for word in sentence.split()
    capitalized_words.append(word.title())
print(" ".join(capitalized_words))